db/migrations/postgres/000012_create_oauth_access_data.up.sql

```diff
@@ -0,0 +1,35 @@
+CREATE TABLE IF NOT EXISTS oauthaccessdata (
+    token VARCHAR(26) NOT NULL,
+    refreshtoken VARCHAR(26),
+    redirecturi VARCHAR(256),
+    PRIMARY KEY (token)
+);
+
+CREATE INDEX IF NOT EXISTS idx_oauthaccessdata_refresh_token ON oauthaccessdata (refreshtoken);
+
+ALTER TABLE oauthaccessdata ADD COLUMN IF NOT EXISTS clientid VARCHAR(26) DEFAULT '';
+CREATE INDEX IF NOT EXISTS idx_oauthaccessdata_client_id ON oauthaccessdata (clientid);
+
+ALTER TABLE oauthaccessdata ADD COLUMN IF NOT EXISTS userid VARCHAR(26) DEFAULT '';
+CREATE INDEX IF NOT EXISTS idx_oauthaccessdata_user_id ON oauthaccessdata (userid);
+
+DO $$
+BEGIN
+    IF NOT EXISTS (
+        SELECT conname
+        FROM pg_constraint
+        WHERE conname = 'oauthaccessdata_clientid_userid_key'
```

Comment:
Not sure if this is the best way to fetch the unique constraint as its not linked to any table, but name is unique and it works. Just in case you know a better way to do it

Reply:
It looks like this is part of the gorp create table, so i guess we can added in `CREATE TABLE ...`